tests/assembly/module-features.wat

```diff
@@ -8,5 +8,6 @@
   local.get $0
   local.get $1
   i32x4.add
+  i8x16.relaxed_swizzle
```

Comment:
This line will be overwritten when the .wat file is rebuilt. Relaxed SIMD isn't in AssemblyScript yet, though, so that can't be prevented. Perhaps add a comment here so future commits don't accidentally delete this line before there is a proper way to emit the instruction?

Reply:
Good point. Relaxed SIMD is implemented in wabt so I just regenerated the wast file.

Reply:
For now this will be fine I think, that is as long as nobody runs [these](https://github.com/AssemblyScript/wabt.js/blob/94590d71104f126df5b15ebfbad0c069cf1104a3/package.json#L28-L30) (CI doesn't). Either with a comment so we don't forget, or we could decide right away to remove the AS sources and the build tasks and rely solely on .wat files?

Reply:
I see, in that case I think it's best to remove the relaxed_simd stuff. I will just leave the tests as they are. The simd flag is now optional so it can be removed it from the tests (but it wouldn't harm to leave it in either).

Reply:
Sure, that works as well